utils/zerovec/src/cow.rs

```diff
@@ -97,7 +97,7 @@ impl<'a, V: VarULE + ?Sized> VarZeroCow<'a, V> {
     }
 
     /// Construct from an owned slice. Errors if the slice doesn't represent a valid `V`
-    pub fn parse_owned_byte_slice(bytes: Box<[u8]>) -> Result<Self, UleError> {
+    pub fn parse_owned_bytes(bytes: Box<[u8]>) -> Result<Self, UleError> {
```

Comment:
Suggestion (optional): `try_from_boxed_bytes` or `parse_boxed_bytes`. I think "boxed bytes" is language used elsewhere

Reply:
I think for a Cow type "owned" is probably better.